Service/NotificationManager.php

```diff
@@ -0,0 +1,170 @@
+<?php
+  /**
+   * Created by groovili
+   * https://github.com/groovili
+   */
+  declare(strict_types = 1);
+
+  namespace Kopay\NotificationBundle\Service;
+
+  use Kopay\NotificationBundle\Entity\NotificationEmail;
+  use Kopay\NotificationBundle\Entity\NotificationMessageInterface;
+  use Kopay\NotificationBundle\Entity\NotificationPush;
+  use Symfony\Component\OptionsResolver\Exception\InvalidArgumentException;
+
+  /**
+   * Class NotificationManager
+   *
+   * @package Kopay\NotificationBundle\Service
+   */
+  class NotificationManager {
```

Comment:
What does this class actually manage? 
This class violates SRP - it behaves like a factory for notifications, it sends something and it provides a fluent interface to create notifications - this should be at least 3 separate classes - NotificationFactory, NotificationBuider and NotificationSender.